packages/core/src/menus/items/_types/IStandardMenuItemData.ts

```diff
@@ -12,4 +14,6 @@ export type IStandardMenuItemData = IStandardActionBindingData & {
         | IThemeIcon
         | ReactElement
         | ((h?: IDataHook) => IThemeIcon | ReactElement | undefined);
+    /** The text-highlighter used to highlight parts of the name/description when this item is found with a search, or null to skip highlighting */
+    TextHighlighter?: LFC<ISearchHighlighterProps> | null;
```

Comment:
Again any reason why null is allowed?

Reply:
Yes, and here you see that it actually is documented.

Reply:
I didn't see the comment xD oops.